manifests/q/QTTabBar/QTTabBar/1.5.260/QTTabBar.QTTabBar.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: QTTabBar.QTTabBar
+PackageVersion: 1.5.260
+PackageLocale: en-US
+Publisher: QTTabBar

```

```suggestion
Publisher: Quizo and Paul Accisano
```
This must match the value in control panel